### DISCUSSION TITLE
:Gclog quickfix list locked until ctrl-c

### DISCUSSION THREAD

**Discussion by blissdev** [tpope/vim-fugitive#1845] [Q&A] [answered]
After running `:Gclog` the quickfix list with commits comes up, but vim stops taking any inputs. To be able to interact again I seem to have to use ctrl+c. Is there something I'm missing or maybe this is just a neovim problem?

```
NVIM v0.5.0
Build type: Release
LuaJIT 2.1.0-beta3
Compilation:
Compiled by _nixbld1

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/nix/store/n281kyv3cryc4cr8fnl4sm42zi6rwic0-neovim-unwrapped-0.5.0/share/nvim
"

Run :checkhealth for more info
```

**Comment by tpope** [comment, marked as answer]
If the history is long then what you're missing is this:

https://github.com/tpope/vim-fugitive/blob/142a0dc0c472016988d3fc76d055f4b00bf783f1/doc/fugitive.txt#L118-L120
